app/indexers/concerns/oregon_digital/indexes_basic_metadata.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module OregonDigital
```

Comment:
The changes in this PR removed references to IndexesBasicMetadata and IndexesLinkedMetadata, so do we still need these overrides?
If so, since these are under the OregonDigital module, wouldn't the references in Hyrax not see these overrides unless they're in the Hyrax module?